frontend/app/src/store/defi/const.ts

```diff
@@ -96,18 +114,28 @@ export const OVERVIEW_PROTOCOLS = [
   CHAI,
   IEARNFINANCEV3,
   IEARNFINANCEV2,
-  IDLE_BESTYIELD,
+  IDLE,
+  IDLE_RISKADJUSTED,
   DYDX,
   CURVE,
+  CURVE_VESTING,
+  CURVE_LIQUIDITY_GAUGES,
   COMPOUND,
+  COMPOUND_GOVERNANCE,
   BALANCER,
   AAVE,
+  AAVE_STAKING,
+  AAVE_UNISWAPMARKET,
   MAKERDAO
 ] as const;
 
 const ICON_LIST: { [protocol in OverviewDefiProtocol]: string } = {
+  [ZLOTFINANCE]: '',
+  [YGOVFINANCEV1]: '',
+  [YGOVFINANCEV2]: '',
```

Comment:
Kelsos can you please find and add svgs for this? Also I think this BEREZKA thing does not have an icon. Perhaps just do it in another PR since this should go in develop to fix the test